frontend/javascripts/oxalis/model/bucket_data_handling/bucket.ts

```diff
@@ -593,12 +611,11 @@ export class DataBucket {
 
     switch (this.state) {
       case BucketStateEnum.REQUESTED: {
-        // Clone the data for the unmergedBucketDataLoaded event,
-        // as the following merge operation is done in-place.
-        const dataClone = new TypedArrayClass(data);
```

Comment:
this is unrelated to the actual PR. the cloning of data was always done even though it's only necessary before doing a merge.